server/command.go

```diff
@@ -320,7 +356,7 @@ func (p *Plugin) executeSettingsCommand(c *plugin.Context, args *model.CommandAr
 		Message: p.b.LocalizeWithConfig(l, &i18n.LocalizeConfig{
 			DefaultMessage: &i18n.Message{
 				ID:    "jitsi.command.settings.updated",
-				Other: "Jitsi settings updated",
+				Other: fmt.Sprintf("Jitsi settings updated:\n\n* %s: `%s`", parameters[0], parameters[1]),
```

Comment:
```suggestion
				Other: fmt.Sprintf("Jitsi settings updated:\n\n %s: `%s`", parameters[0], parameters[1]),
```
What's the need of `*` here? Also, the values you're printing seem wrong.

Reply:
It's a markdown for bullet mark:
![Screenshot from 2022-12-13 15-37-07](https://user-images.githubusercontent.com/55234496/207289001-a847cb03-00ad-42b8-8c32-fea0e66d1135.png)